api/core/television/television.sendCommand.js

```diff
@@ -0,0 +1,28 @@
+const Promise = require('bluebird');
+const queries = require('./television.queries.js');
+
+module.exports = function sendCommand(functionName, params) {
+    return getDeviceType(params)
+      .then((deviceType) => {
+
+          if(!gladys.modules[deviceType.service] || !gladys.modules[deviceType.service].television)
+                return Promise.reject(new Error(`television : Module ${deviceType.service} does not exist or does not handle television.`));
+         
+          if(typeof gladys.modules[deviceType.service].television[functionName] != 'function') {
+            return Promise.reject(new Error(`television : Module ${deviceType.service} does not have function ${functionName}`));
+          } 
+
+          params.deviceType = deviceType;
+        
+          // calling television module
+          return gladys.modules[deviceType.service].television[functionName](params);
+        });
+};
+
+function getDeviceType(params){
+    if(params.device.id) return gladys.utils.sqlUnique(queries.getDeviceTypeByDeviceId, [params.device.id]);
```

Comment:
Just one latest feedback on this! 

In the music API, when we can the service, we do: 

```
gladys.music.play({ devicetype: 1 });
```

Here, with your implementation, we should do : 

```
gladys.music.play({ device: { id: 1} });
```

Not really consistent! 

Can we move to :

```
gladys.music.play({ device: 1 });
```
 ?

Thanks!

Reply:
Changing on commit a4f0173